components/suggest/src/suggest.udl

```diff
@@ -38,7 +38,8 @@ interface Suggestion {
         string iab_category,
         string impression_url,
         string click_url,
-        string raw_click_url
+        string raw_click_url,
+        f64? score
```

Comment:
Since we always use the default score now, I wonder if we should make this non-optional. WDYT?

Reply:
On further thought, let's make this non-optional—there's no need to keep it nullable if we always set its value.